compiler/rustc_target/src/spec/x86_64_unknown_none_elf.rs

```diff
@@ -0,0 +1,31 @@
+// Generic AArch64 target for bare-metal code - Floating point disabled
+//
+// Can be used in conjunction with the `target-feature` and
+// `target-cpu` compiler flags to opt-in more hardware-specific
+// features.
+//
+// For example, `-C target-cpu=cortex-a53`.
+
+use super::{CodeModel, LinkerFlavor, LldFlavor, PanicStrategy, Target, TargetOptions};
+
+pub fn target() -> Target {
+    let opts = TargetOptions {
+        abi: "softfloat".to_string(),
+        linker_flavor: LinkerFlavor::Lld(LldFlavor::Ld),
+        linker: Some("rust-lld".to_owned()),
+        features: "-mmx,-sse,+soft-float".to_string(),
```

Comment:
```suggestion
        features: "-mmx,-sse,-sse2,-sse3,-ssse3,-sse4.1,-sse4.2,-3dnow,-3dnowa,-avx,-avx2,+soft-float".to_string(),
```

(Taken from `x86_64-unknown-none-linuxkernel`.)